types/mkdirp-promise/index.d.ts

```diff
@@ -1,13 +0,0 @@
-// Type definitions for mkdirp-promise 5.0
```

Comment:
I think that deleting typings for this package is unnecessary, because they can continue to be used (600k downloads per week).

Reply:
This is a great topic for conversation :) I removed it based on the conversation here:
https://github.com/microsoft/types-publisher/pull/736

If I don't remove these types, we would need to make changes to them to make this change work.  `@types/mkdirp-promise` assumes that it can use `@types/mkdirp` without pinning to any particular semver range.  When making this change and keeping it around, it caused build failures. 

The alternative to this approach (I think..) is to introduce a package.json to `@types/mkdirp-promise`, and pin that to the `~0.5.0` range.  I started down that path, which led to the whole whitelist thing above and well...  yeah, I followed the advice in that thread :)  The part that swayed me was knowing that a.) updates to `@types/mkdirp-promise` are unlikely at this stage, and b.) the package will still exist - it won't get taken down or anything.

All that having been said - I'm super flexible here!

Reply:
Also, we can restore the mkdir-promise source to the DT repo at any time in case we need to publish a new version &mdash; although the PR that did that would still need to introduce a package.json etc etc.